Resources/translations/SonataClassificationBundle.fr.xliff

```diff
@@ -12,7 +12,7 @@
             </trans-unit>
             <trans-unit id="label_tags">
                 <source>label_tags</source>
-                <target>Tags</target>
+                <target>Étiquettes</target>
```

Comment:
I know "étiquette" is french and "tag" isn't, but in this context I think french people are more used to seeing "tag". This would maybe make sense in an `fr_CA` traduction, along with "courriel" or "patin à roulettes".